tests/golos.publication_tests.cpp

```diff
@@ -559,4 +583,47 @@ BOOST_FIXTURE_TEST_CASE(upvote_near_close, golos_publication_tester) try {
     BOOST_CHECK_EQUAL(err.upvote_near_close, vote_chucknorris(cfg::_100percent));          // TODO Fix broken test GolosChain/golos-smart#410
 } FC_LOG_AND_RETHROW()
 
+BOOST_FIXTURE_TEST_CASE(set_curators_prcnt, golos_publication_tester) try {
+    BOOST_TEST_MESSAGE("Test curators percent");
+    
+    init();
+
+    auto ref_block_num = control->head_block_header().block_num();
+    auto create_msg = [&](auto curators_prcnt){ 
+        return post.create_msg(
+            {N(brucelee), "permlink", ref_block_num}, 
+            {N(), "parentprmlnk", 0},
+            0,
+            {},
+            5000,
+            false,
+            "headermssg",
+            "bodymssg",
+            "languagemssg",
+            {{"tag"}},
+            "jsonmetadata",
+            curators_prcnt    
+            );
+    };
+
+    BOOST_TEST_MESSAGE("--- checking that curators percent doesn't fit");
+    BOOST_CHECK_EQUAL(err.cur_prcnt_no_fit, create_msg(-1));
```

Comment:
actually, this check is wrong, because -1 becomes unsigned. combined error message makes things worse because it masks actual error.
test written to check less than `min_curators_prcnt`, but actually checks greater than `max_curators_prcnt`. please check default `min_curators_prcnt` to 10% or so and check here 10%-1